docs/exploration.md

```diff
@@ -14,10 +14,27 @@ This page gives an overview of ***Avail Uncharted***. It shows the exploration w
 |:------------------------- | :----- | :-------- | 
-| op-evm                  |  OpEVM is a sovereign EVM-compatible optimistic rollup construction designed for efficient and secure transaction processing  |  [link](https://github.com/availproject/op-evm)  |
-| sovereign-da-adapter  |  Data availability adapter for Sovereign Rollups integrated with sovereign-sdk   | [link](https://github.com/availproject/sovereign-sdk/tree/main) |
-| madara-da-interface | A unified DA interface for Madara Starknet Sequencer to publish data onto a preferred DA Layer such as Avail for a reliable and secure DA    | [link](https://github.com/keep-starknet-strange/madara/pull/1021) | 
-| validium-node and validium-contracts |A production-ready validium based on the Polygon zkEVM stack with Avail as the data availability layer for scalable and secure DA | [link (node)](https://github.com/QEDK/validium-node) <br/>&<br/> [link (contracts)](https://github.com/QEDK/validium-contracts) | 
-| avail-op-stack-adapter    | Avail DA's integration with Optimism's rollup SDK op-stack | [link](https://github.com/availproject/avail-op-stack-adapter) | 
-| rollkit-da-adapter  | A Data availability adapter for Rollkit's modular rollup framework | [link](https://github.com/rollkit/rollkit/pull/1168) | 
+| OpEVM   |  OpEVM is a sovereign EVM-compatible optimistic rollup construction designed for efficient and secure transaction processing  |  [OpEVM](https://github.com/availproject/op-evm)  |
+| Adapter for Soverign SDK  | A data availability adapter for Sovereign Rollups integrated with sovereign-sdk   | [sovereign-da-adapter](https://github.com/availproject/sovereign-sdk/tree/main) |
+| Madara DA Interface | A unified DA interface for Madara Starknet Sequencer to publish data onto a preferred DA Layer such as Avail for a reliable and secure DA    | [madara-da-interface](https://github.com/keep-starknet-strange/madara/pull/1021) | 
+| Validium Node and Validium Contracts |A production-ready validium based on the Polygon zkEVM stack with Avail as the data availability layer for scalable and secure DA | [validium-node](https://github.com/QEDK/validium-node) <br/>&<br/> [validium-contracts](https://github.com/QEDK/validium-contracts) | 
+| Avail Stack Adapter    | Avail DA's integration with Optimism's rollup SDK op-stack | [avail-op-stack-adapter ](https://github.com/availproject/avail-op-stack-adapter) | 
+| Rollkit DA Adapter  | A Data availability adapter for Rollkit's modular rollup framework | [rollkit-da-adapter ](https://github.com/rollkit/rollkit/pull/1168) | 
 
+## OpEVM
+OpEVM is an EVM-compatible sovereign optimistic rollup SDK with working fraud proofs and has support for a decentralized sequencer set. OpEVM is designed for efficient and secure transaction processing. It provides a decentralized infrastructure for running a layer-2 (L2) blockchain. OpEVM enables high-throughput, low-latency transaction processing.
```

Comment:
We don't mention anywhere how this is built on top of Avail and briefly how it works with Avail.